compositions/use-gamepad-button-pushed.js

```diff
@@ -50,8 +54,27 @@ export function useGamepadButtonPushed () {
     start = false
   }
 
-  function gameLoop () {
+  /**
+   * Retrieves the gamepads of navigator as a regular array, ensuring compatibility
+   * with functional additions like map and filter.
+   */
+  function retrieveGamepads () {
     const gamepads = navigator.getGamepads()
+    if (!gamepads) {
+      return undefined
+    }
+
+    const result = []
```

Comment:
```suggestion
    return Array.from(gamepads) // Provide gamepads as array to ensure cross-browser compatibility
```

instead of the rest of the function.

(unfortunately GitHub doesn' support multiline suggestions yet)

Reply:
Array.from is much sleeker and cleaner - thanks, much appreciated! 👍